crates/nu-command/src/viewers/table.rs

```diff
@@ -18,6 +18,16 @@ use terminal_size::{Height, Width};
 const STREAM_PAGE_SIZE: usize = 1000;
 const STREAM_TIMEOUT_CHECK_INTERVAL: usize = 100;
 
+fn get_width_param(width_param: Option<i64>) -> usize {
```

Comment:
As `crates/nu-command/src/viewers/griddle.rs` do.  
In https://github.com/nushell/nushell/blob/main/crates/nu-command/src/viewers/griddle.rs#L216=.